### PR TITLE
🔗 Refer to MS learn instructions

### PR DESCRIPTION
**Pull request by bradystroud**
- 🔧 Less maintenance needed on this rule - MS will keep their docs up to date so we don't have to
- 🧼 Shorter, cleaner rule

https://learn.microsoft.com/en-us/connectors/custom-connectors/port-tunneling

### CODE REVIEW

File: rules/port-forwarding/rule.md [hunk 1]
```diff
@@ -39,36 +39,7 @@ Luckily, port forwarding is the silver...errr...golden bullet that solves this i
 
 ## Dev Tunnels Setup
```
Comment (SamBWagner) [inline rules/port-forwarding/rule.md:40]: It feels like we're not just sending them to the docs but rather deleting the whole section. Either keep all or lose all

**Review by SamBWagner — CHANGES_REQUESTED** [review]
Left comments to address :) Feels like we're deleting a lot.

**Review by SamBWagner — APPROVED** [review]
🚢
Checked by Bryden

**Review by danielmackay — CHANGES_REQUESTED** [review]
I like the idea of leveraging the MS learn docs, however I feel the step outlined in the rule are a lot more streamlined compared to MS learn (which covers APIM and Power Platform, but also less details about the core use case).

My preference would be to leave the screenshots and instructions in the rule.

**Comment by SamBWagner** [comment]
I believe the sentiment was that providing a guide, if it's already in the docs, is now a maintenance cost.

**Comment by adamcogan** [comment]
Hey @danielmackay @bradystroud @SamBWagner @lukecookssw @pierssinclairssw 

I agree that in general SSW Rules should not create content that exists elsewhere eg. docs.microsoft.com

However in this case:
A: the Rule is clearer and shorter than the reference docs.

B: this is a new feature and so this SSW Rule may end up getting more links from other sites, since it is simpler to read.

C: the Rule is already created and at this stage it is not out-of-date. So this would be pre-optimizing to kill the content.

In short, kill the content when it is out-of-date... and the effort to update it is not worth it.

-a
